### PR TITLE
[zh-cn]: add the translation of iterator.drop()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

As the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#17677

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: Iterator.prototype.drop()
+slug: Web/JavaScript/Reference/Global_Objects/Iterator/drop
+l10n:
+  sourceCommit: 088b56a895d22b6df854a9f26400af7d399f289f
+---
+
+{{JSRef}}{{SeeCompatTable}}
+
+{{jsxref("Iterator")}} 实例的 **`drop()`** 方法返回一个新的[迭代器辅助方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Iterator#迭代器辅助方法)，该迭代器辅助方法会跳过该迭代器的开头的给定数量的元素。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md:10]:
```suggestion
{{jsxref("Iterator")}} 实例的 **`drop()`** 方法返回一个新的[迭代器辅助方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Iterator#迭代器辅助方法)，该迭代器辅助方法会跳过此迭代器的开头的给定数量的元素。
```

File: files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+### 参数
+
+- `limit`
+  - : 要从迭代器的开头丢弃的元素数量。
+
+### 返回值
+
+一个新的[迭代器辅助方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Iterator#iterator_helpers)。第一次调用返回的迭代器辅助方法的 `next()` 方法时，当前迭代器立即前进 `limit` 个元素，然后生成下一个元素（即第 `limit+1` 个元素）。然后，迭代器辅助方法会逐一生成剩余元素。如果当前迭代器的元素数量少于 `limit` 个，那么新生成的迭代器辅助方法会在 `next()` 方法第一次调用时立即完成。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md:25]:
```suggestion
一个新的[迭代器辅助方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Iterator#迭代器辅助方法)。第一次调用返回的迭代器辅助方法的 `next()` 方法时，当前迭代器立即前进 `limit` 个元素，然后生成下一个元素（即第 `limit+1` 个元素）。然后，迭代器辅助方法会逐一生成剩余元素。如果当前迭代器的元素数量少于 `limit` 个，那么新生成的迭代器辅助方法会在 `next()` 方法第一次调用时立即完成。
```

File: files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+### 异常
+
+- {{jsxref("RangeError")}}
+  - : 如果 `limit` [转换为整数后](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#integer_conversion) 变成 {{jsxref("NaN")}} 或负数，则抛出该异常。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md:30]:
```suggestion
  - : 如果 `limit` [转换为整数后](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换) 变成 {{jsxref("NaN")}} 或负数，则抛出该异常。
```

File: files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  - : 要从迭代器的开头丢弃的元素数量。
+
+### 返回值
+
+一个新的[迭代器辅助方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Iterator#迭代器辅助方法)。第一次调用返回的迭代器辅助方法的 `next()` 方法时，当前迭代器立即前进 `limit` 个元素，然后生成下一个元素（即第 `limit+1` 个元素）。然后，迭代器辅助方法会逐一生成剩余元素。如果当前迭代器的元素数量少于 `limit` 个，那么新生成的迭代器辅助方法会在 `next()` 方法第一次调用时立即完成。
+
+### 异常
+
+- {{jsxref("RangeError")}}
+  - : 如果 `limit` [转换为整数后](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换) 变成 {{jsxref("NaN")}} 或负数，则抛出该异常。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/iterator/drop/index.md:30]:
```suggestion
  - : 如果 `limit` [转换为整数后](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)变成 {{jsxref("NaN")}} 或负数，则抛出该异常。
```

**Review by yin1999 — APPROVED** [review]
Thank you @JasonLamv-t

**Pull request merged**